packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerCorrectness.test.ts

```diff
@@ -665,28 +661,34 @@ export function testCorrectness() {
 		 * - They help diagnose issues with the more complicated exhaustive test (e.g., if one of the above tests fails,
 		 * but this one doesn't, then there might be something wrong with this test).
 		 */
-		it("Combinatorial test", () => {
+		describeStress("Combinatorial exhaustive", ({ isStress }) => {
+			const NUM_STEPS = isStress ? 5 : 4;
+			const NUM_PEERS = 2;
```

Comment:
Would that be too much?
```suggestion
			const NUM_PEERS = 3;
```

Reply:
Nope, makes it take about as long as some other suites we have (17s on my machine, longest other suite is 21s).